.github/workflows/release-server-sdk.yml

```diff
@@ -0,0 +1,93 @@
+name: 'Release Server SDK'
+on:
+  workflow_call:
+    inputs:
+      language:
+        description: 'Programming language for the project. Supported values are "java", "dotnet", "python", "golang", and "php".'
+        required: true
+        type: string
+      language-version:
+        description: 'Version of the programming language to setup.'
+        required: true
+        type: string
+      java-version:
+        description: 'Version of Java to setup.'
+        required: false
+        type: string
+        default: '11'
+      prepare-command:
+        description: 'Command to run for project preparation (e.g., installing dependencies).'
+        required: true
+        type: string
+      semantic-release-extra-plugins:
+        description: 'Additional plugins to install for the semantic-release action.'
+        required: false
+        type: string
+    secrets:
+      GH_RELEASE_TOKEN:
+        description: 'GitHub token with permissions to create releases and perform other necessary operations.'
+        required: true
+      PYPI_TOKEN:
+        description: 'PyPI token used for publishing Python packages. Required only for Python projects.'
+        required: false
+      NUGET_API_KEY:
+        description: 'NuGet API key for publishing .NET packages. Required only for DotNET projects.'
+        required: false
```

Comment:
Shouldn't these be passed as secrets instead?

Reply:
It is